utils/auth.py

```
@@ -0,0 +1,69 @@
+    try:
+        print("[hash_password] len(chars):", len(password))
+        print("[hash_password] len(bytes):", len(password.encode("utf-8")))
+        print("[hash_password] preview:", password[:120])
+    except Exception as e:
+        print("[hash_password] not a str:", password, e)
+    return pwd_context.hash(password)
+
+def verify_password(password: str, hashed_password: str) -> bool:
+    return pwd_context.verify(password, hashed_password)
+
+def create_access_token(subject: str, expired_in = ACCESS_TOKEN_EXPIRE_MINUTES) -> str:
```

Ruff (0.14.13)

Unused function argument: `expired_in`

(ARG001)